<?xml version="1.0" encoding="utf-8"?>
<launch>
	<!-- <param name="result_file_name" type="string" value="result_k_0.5.csv" /> -->
<!-- 	<rosparam command="load" file="$(find baxter_trajectory_interface)/config/baxter_param_mod.yaml" />
	<rosparam command="load" file="$(find baxter_trajectory_interface)/config/velocity_ffController_param.yaml" /> -->
	<!-- <rosparam command="load" file="$(find baxter_trajectory_interface)/config/vm_param.yaml" /> -->
	<!-- 	<rosparam command="load" file="$(find baxter_trajectory_interface)/config/vm_steering.yaml" /> -->
	<!-- <node pkg="baxter_trajectory_interface" type="controller_dq" name="controller_dq" output="screen" launch-prefix="valgrind" /> -->
<!-- 	<node pkg="tf2_ros" type="static_transform_publisher" name="ar10_right_palm2right_handBaxter" args="0 0 0 0 0 -0.7071068 0.7071068 right_hand ar10_right/wrist_plate" />
	<node pkg="tf2_ros" type="static_transform_publisher" name="ar10_right_ciruitSupport2ar10_right_origin" args="0 0 0.012 0 0 0 1 ar10_right/wrist_plate ar10_right/circuit_support" />	 -->
	<rosparam command="load" file="$(find dq_robotics)/config/finger_params.yaml" />
	<rosparam command="load" file="$(find dq_robotics)/config/controller_param.yaml" />
	<rosparam command="load" file="$(find dq_robotics)/config/velocity_ffController_param.yaml" />
	<rosparam command="load" file="$(find dq_robotics)/config/baxter_param_mod.yaml" />	
	<!-- <node pkg="dq_robotics" type="ar10Kinematics" name="ar10Kinematics" output="screen" />	 -->
	<node pkg="dq_robotics" type="baxter_ar10_kinematics" name="baxter_ar10_kinematics" output="screen" />
	<node pkg="dq_robotics" type="rviz_shape_publisher" name="rviz_shape_publisher" output="screen" />
<!-- 	<rosparam command="load" file="$(find dq_robotics)/config/velocity_ffController_param.yaml" />
	<node pkg="dq_robotics" type="baxter_AbsRelative_Control" name="baxter_AbsRelative_Control" output="screen" />  -->
</launch>
